frontend/src/p2/NextSteps.js

```diff
@@ -12,107 +13,142 @@ import { Layout } from '../components/layout'
+                    <strong>Where you can get help</strong>
+                  </Trans>
+                </P>
+                <Ul>
+                  <Li>
+                    <A
+                      href={
+                        language === 'en'
+                          ? 'http://www.rcmp-grc.gc.ca/ccaps-spcca/vic-eng.htm'
+                          : 'http://www.rcmp-grc.gc.ca/ccaps-spcca/vic-fra.htm'
+                      }
+                    >
+                      <Trans>Talk to someone and get support</Trans>
+                    </A>
+                  </Li>
+                  <Li>
+                    <A
+                      href={
+                        language === 'en'
+                          ? 'http://www.rcmp-grc.gc.ca/to-ot/tis-set/cyber-tips-conseils-eng.htm'
+                          : 'http://www.rcmp-grc.gc.ca/to-ot/tis-set/cyber-tips-conseils-fra.htm'
+                      }
+                    >
+                      <Trans>Find tips to protect yourself in the future</Trans>
+                    </A>
+                  </Li>
+                  <Li>
+                    <A
+                      href={
+                        language === 'en'
+                          ? 'http://www.antifraudcentre.ca/index-eng.htm'
+                          : 'http://www.antifraudcentre.ca/index-fra.htm'
+                      }
+                    >
+                      <Trans>Follow what scams are trending</Trans>
+                    </A>
+                  </Li>
+                </Ul>
+                <P>
+                  <Trans>
+                    <strong>What happens next</strong>
+                  </Trans>
+                </P>
+                <P>
+                  <Trans></Trans>
+                </P>
+                <P>
+                  <Trans>
+                    The RCMP will analyze your report and combine it with others
+                    that are similar to help police across Canada track down
+                    cybercriminals.
+                  </Trans>
+                </P>
+              </InfoCard>
+              <br></br>
+              <P>
+                <Trans>
+                  <strong>You have other reporting options</strong>
+                </Trans>
+              </P>
+              <Ul>
+                <Li>
+                  <Trans>
+                    Report directly to the place where the scam happened.
+                  </Trans>
+                </Li>
+                <Li>
+                  <Trans>
+                    Report to your financial institution, if you lost money.
+                  </Trans>
+                </Li>
+                <Li>
+                  <Trans>
+                    Report to the credit bureaus,{' '}
+                    <A href="https://www.consumer.equifax.ca/personal/">
+                      Equifax
+                    </A>{' '}
+                    and <A href="https://www.transunion.ca/">TransUnion</A>, if
```

Comment:
Equifax and TransUnion also have French pages.

Reply:
ah, I made a mental note about those but then forgot! Thanks Steve-o